### ISSUE TITLE
nwwin.isDevToolsOpen is not a function

### ISSUE THREAD

**Issue by royaldark** [1j01/nw-dev#3]
Followed the installation instructions, added `node_modules/nw-dev/lib/dev.js` to my project. Immediately upon startup, nw crashes with

```
chrome-extension://clpmodkidojamblhiaibbmdpdmpiojnj/node_modules/nw-dev/lib/dev.js:22
      if (!nwwin.isDevToolsOpen()) {
                 ^

TypeError: nwwin.isDevToolsOpen is not a function
    at process.<anonymous> (chrome-extension://clpmodkidojamblhiaibbmdpdmpiojnj/node_modules/nw-dev/lib/dev.js:22:18)
```

I'm running nw version 51.0.2704.29 on Ubuntu 15.10.

**Comment by 1j01** [comment]
You mean nw 0.15.0 beta? You'll need the "SDK version" for devtools, i.e. `npm i nw@0.15.0-beta2sdk-2`, although I haven't tested this version yet, so the API could be removed for all I know.

**Comment by royaldark** [comment]
Yes, 0.15.0-beta2. Not sure why the output of `nw --version` is different.

I am using the SDK version, although I wonder if this [issue I filed upstream](https://github.com/nwjs/nw.js/issues/4819) is the culprit.

I guess this is what I get for choosing a beta version my first time using nw :)

**Comment by 1j01** [comment]
The API method is apparently removed.
I've published `nw-dev@2.0.0` which should work for the most part with 0.15.0.
